ui/lib/open-api-explorer/addon/templates/components/swagger-ui.hbs

```diff
@@ -35,4 +35,4 @@
     for examples.
   </NamespaceReminder>
   <div id="{{this.elementId}}-swagger" class="swagger-ember"></div>
-</div>
+</div>
```

Comment:
Was this a linting thing?

Reply:
yeah, so strange. It caused my tests to fail first round.